src/ontology/mondo-ingest.Makefile

```diff
@@ -13,6 +13,28 @@
 MAPPINGSDIR=../mappings
 SKIP_HUGE=false
 
+####################################
+### Package management #############
+####################################
+.PHONY: pip
+pip:
+	python3 -m pip install --upgrade pip setuptools
+
+.PHONY: oaklib
+oaklib: pip
+	 python3 -m pip install --upgrade $@
+
+.PHONY: sssom
+sssom: pip
+	python3 -m pip install --upgrade $@
+
+.PHONY: semsql
+semsql: pip
+	python3 -m pip install --upgrade semsql>=0.3.2
+
+.PHONY: dependencies
+dependencies: oaklib sssom semsql
```

Comment:
Previously was only installing `semsql` and `oaklib`.
Currently no goal depends on this. I think it is just for ad hoc use.

Reply:
Exactly, you can use it like this:

```
sh run.sh make dependencies whateverothergoalyouneed
```